maltoolbox/language/languagegraph.py

```diff
@@ -567,10 +538,26 @@ def _from_dict(cls,
                 assoc_name = list(serialized_expr_chain.keys())[0]
                 target_asset = lang_graph.assets[\
                     serialized_expr_chain[assoc_name]['asset type']]
+                fieldname = serialized_expr_chain[assoc_name]['fieldname']
+
+                association = None
+                for assoc in target_asset.associations.values():
+                    if assoc.contains_fieldname(fieldname) and \
+                            assoc.name == assoc_name:
+                        association = assoc
+                        break
+
+                if association is None:
+                    msg = 'Failed to find association "%s" with '\
+                        'fieldname "%s"'
+                    logger.error(msg % (assoc_name, fieldname))
+                    raise LanguageGraphException(msg % (assoc_name,
+                        fieldname))
```

Comment:
```suggestion
                for assoc in target_asset.associations.values():
                    if assoc.contains_fieldname(fieldname) and \
                            assoc.name == assoc_name:
                        association = assoc
                        break
                else:
                    msg = 'Failed to find association "%s" with '\
                        'fieldname "%s"'
                    logger.error(msg % (assoc_name, fieldname))
                    raise LanguageGraphException(msg % (assoc_name,
                        fieldname))
```

Reply:
I do not like the for-break-else format.